<launch>

    <node pkg="world_cloud_converter" type="cloud_converter_cuda_node" name="" output="screen">
        <param name="ed_left" value="7.7175"/>              <!-- 左侧边界 -->
        <param name="ed_right" value="-0.2825"/>            <!-- 右侧边界 -->
        <param name="ed_ceil" value="14.7175"/>             <!-- 顶部边界 -->
        <param name="ed_floor" value="-0.2825"/>            <!-- 底部边界 -->
        <param name="ed_resolution" value="0.2"/>           <!-- 网格分辨率 -->
        <param name="yaw_1" value="-0.785398163" />         <!-- 1号雷达坐标系与车体坐标系的偏差 -->
        <param name="yaw_2" value="0.785398163" />          <!-- 2号雷达坐标系与车体坐标系的偏差 -->
        <param name="dist1_x" value="0.25301" />            <!-- 1号雷达在车体坐标系下的坐标 -->
        <param name="dist1_y" value="-0.25301" />
        <param name="dist2_x" value="0.25301" />            <!-- 2号雷达在车体坐标系下的坐标 -->
        <param name="dist2_y" value="0.25301" />
        <param name="half_height" value="0.2825" />            <!-- 车体高度的一半 -->
        <param name="half_width" value="0.2825" />             <!-- 车体宽度的一半 -->
        <param name="cloud_topic_1" value="/cloud_right" /> <!-- 1号雷达的点云话题 -->
        <param name="cloud_topic_2" value="/cloud_left" />  <!-- 2号雷达的点云话题 -->
    </node>

</launch>